### PR TITLE
UCT/IB: Fix DC test for mlx5dv, when DevX is not available

### PR DESCRIPTION
**Pull request by yosefe**
# Why

Before this fix DC was disabled on RoCE interface because the internal check for DC support did not set is_global=1 for INIT2RTR.

# How
Set is_global=1 and add debug prints

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 3 of 23 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/67141485de2a3da642ed5fcce197f0ff4550dfe8/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/614dc4fadffb2cacc62c0d2c050d1e9a4344fdc9/hpc-arm-cavium-jenkins_W0) | :question: ABORTED
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/a8f31678a49678d2808e79f4c6c2c4ce358659b5/hpc-arm-cavium-jenkins_W1) | :question: ABORTED
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/52a9b1c20ddc6bf4535512bfaf560c6c53871d99/hpc-arm-cavium-jenkins_W2) | :question: ABORTED
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/bfadbe3c50f22c99464451ce4966f885b1769b10/hpc-arm-cavium-jenkins_W3) | :question: ABORTED
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/96a1fe2a4054dead82ccf4c2b4632ff89b51146b/hpc-arm-hwi-jenkins_W0) | :question: ABORTED
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/c201762217cf01d72476d084dced89c2329c83c3/hpc-arm-hwi-jenkins_W1) | :question: ABORTED
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/3d40edc1f147f7db16e47403194a7fa2c306ec93/hpc-arm-hwi-jenkins_W2) | :question: ABORTED
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/739a1d0b0e5986519a89bcce9226af0d2b9e9289/hpc-arm-hwi-jenkins_W3) | :question: ABORTED
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/b6a165f6d7156a3d19b276e6f285d0c341b3b16f/hpc-test-node-gpu_W0) | :question: ABORTED
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/9d8c848bd196e3712e1756587292aa97fa745c3a/hpc-test-node-gpu_W2) | :question: ABORTED
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/b8f7a0836bb33971ac6b8f620c94d65c3b76dc04/hpc-test-node-legacy_W1) | :question: ABORTED
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/8cc63e9fd3d841bb171f96df4d441da20f86398a/hpc-test-node-new_W0) | :question: ABORTED
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/af6ad4fa76e48fd9e9d3a8f08ea0974d1e8e0097/hpc-test-node-new_W2) | :question: ABORTED
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/90d0a10b968c5097e4038c72053e21985bdb41b4/hpc-test-node-new_W3) | :question: ABORTED
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/cf06c655eabcc61fe393aec19885349fd34ed804/r-vmb-ppc-jenkins_W2) | :question: ABORTED
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/df90063f7d6def7d20367770736b0990cf9ccd2d/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/0a1400329fe8f5ccc7e95c8070b9d0810548b794/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/a6b4921da7a12850fbf1353e4cfc32390b03476d/hpc-test-node-legacy_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/14694ff795688b74a0455ba4dd774da9039fa729/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/d8c090d74386e31e33e3120d3d48dfc4d635985c/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/735340736c5ca9325d132598dcdeaf780a90a7fe/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/53adf3066354a63ae9fdc2ddff2444e2/raw/d4d633aa83ab7163d0ab8c4d7da6dd0df5c6b1fc/hpc-test-node-new_W1) | :question: UNKNOWN

**Comment by yosefe** [comment]
hpc-test-node2 issue with ens2f0
bot:mlx:retest

File: src/uct/ib/mlx5/dv/ib_mlx5dv_md.c [hunk 1]
```diff
@@ -795,20 +798,32 @@ static ucs_status_t uct_ib_mlx5dv_check_dc(uct_ib_device_t *dev)
...
+    attr.path_mtu                  = IBV_MTU_256;
+    attr.ah_attr.port_num          = 1;
+    attr.ah_attr.sl                = 0;
+    attr.ah_attr.is_global         = 1;
+    attr.ah_attr.grh.hop_limit     = 1;
+    attr.ah_attr.grh.traffic_class = 0;
+    attr.ah_attr.grh.sgid_index    = 0;
 
     ret = ibv_modify_qp(qp, &attr, IBV_QP_STATE |
                                    IBV_QP_MIN_RNR_TIMER |
```
Comment (dmitrygx) [inline src/uct/ib/mlx5/dv/ib_mlx5dv_md.c]: should we remove `IBV_QP_MIN_RNR_TIMER` flag? we don't set min RNR timer value

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/26b687db9453c1973275da3698cead3ec331d690/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/dd6e0eb8cd522f8672628c5035c8b22c61d3297b/hpc-arm-cavium-jenkins_W1) | :question: ABORTED
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/931105102fcb452dc0cc69b283453e462127ab89/hpc-arm-hwi-jenkins_W0) | :question: ABORTED
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/4b041b02e78374bf45a1b0eed40739a4a4257baf/hpc-arm-hwi-jenkins_W1) | :question: ABORTED
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/ab0bd7088425e6ab1778f6ca5ed4f6dbf6493c4e/hpc-test-node-gpu_W0) | :question: ABORTED
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/130b1c3ee7475fc5d599ce02fb332ed3231e467d/hpc-test-node-gpu_W2) | :question: ABORTED
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/635f592cb97de805850d755d23ac500f94bf4024/hpc-test-node-legacy_W0) | :question: ABORTED
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/f0756e4e57be76c0b7668390f61d2f8ad3ff27ff/hpc-test-node-legacy_W3) | :question: ABORTED
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/303b6f67fec88687a6c116f04b9fce69657e1846/hpc-test-node-new_W0) | :question: ABORTED
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/4c3048ef60e7e88b7021409ce19faa9355c6c6ea/hpc-test-node-new_W1) | :question: ABORTED
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/21b540f8cb495d45f7ba1ed7d57c2ee6b57b1166/hpc-test-node-legacy_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/c07981aed277a6d0f661f3b648b306134cd96ca4/r-vmb-ppc-jenkins_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/5f84a0cf80a9842ab9ef59f262f37a02d60dbd9a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/c5dfb123afddc7922b26c5c7efc267429015746d/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/9a9944e44e4d5e2f75dd6d18359240688c1730f4/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/b5ff79badc77d56cd15afbb3828baf43029adf79/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/0ef7d15852323d6e61349c643269345a1534218c/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/c4a7538a7ea46fa10c2b9fd40e783e7e55f0d7f5/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/48e047e17edce252a19bc378a5dac61e4a5eb73d/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/e48b2e20fd6f2f73551ac2812382f5fe1e84bcb7/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/9e9b288772d6b774de090f7a72f4dcf11fb8951e/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/5da5ba170b8ca1445a47a455e80d2f8cee46e8e8/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/a9f95b10e865776eb8589941e1b59a54b8360133/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/c67e5ced30d55745ff933aa8b6cab0b6288db634/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/77ee081953596a33192ceb864b29591a/raw/e04a5f1fe0d84efe7ad7f52f97d39954000ce0ce/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 14-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/f40668f844c63f0c26c3e086378bf7ed8eb47400/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/aa7aa9eff06fca829e89088317d3a5a6263c6212/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/a1ebc2f660b614dc148af91139d37a236cc021e0/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/77b8ea2ed2987a2d32498490c689cec7617c0f27/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/fd15022d61d307849c2b69e93e3a97b950b79517/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/ef64edcc2bfe63e42959b8292825cd5e9db81ba8/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/2565062ebb8e266ed4afbbb0aa24805675fafffa/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/ab122563343d4451a4aa6abf333f9a15f0d1b02c/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/f0b65fe08b4400f253c271eb7517042121344681/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/48110260c46c94ac48c1215f815bcc108c3284e0/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/58a6e855726787217490bd4e6a4842f1f7841d4c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/a8d074e143a03fb1e955f485c33269775ce23902/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/9ca95ad91a3c37ff27cee960604195ff7bbd9700/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/62881aff0d36f002751b015f56c64f1f315329af/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/7cd8087ebd7bd00cece2b8524c2119199813d149/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/790e1ca3c307edd5dfe4f7f91fc40469253884f5/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/f5961687c134b6e497a3108df29722e0639a2994/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/9224d62ed2525621ba9c6b2007556f796e083f49/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/3fdb96b831932f370b72e0d0b0c03ec2b8bbc6d9/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/eed2975733bab36a8bbd9232b4d55effd01a6a97/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/2228e48b2ceeebf7a0ccf04443e90537e732ba21/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/ec1e0832f0fb97864fc86f273b8daf2f8bcc8cd9/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/e658fc9cd35df9b4853bc6a8ed76d4eb022b65ee/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/cba64b2cf3a853aec28af31cad35f0cb57a515fc/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3b7282e0ed183a345fe48f6904e7ec38/raw/51e6d6c032043bc6db62b94ac3b4eb6f37e2a6d8/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**